src/repository/index.ts

```diff
@@ -77,7 +85,8 @@ export async function init(orm: Orm): Promise<Repositories> {
   /**
    * Create storage instances
    */
-  const userStorage = new UserStorage(orm);
+  const userModelStorage = new UserModelSequelizeStorage(orm);
```

Comment:
model separation and injection is ok. Now we have problems with naming and path. Looks like now we have to separate things: model and storage. 

1. So UserModel should not be called "storage" because it is not a storage but think used by storages.
2. UserModel should not be placed on the same level with storages.

Reply:
In that case, how about moving the userModel to the repository level?

Reply:
it's not a repository. Let's move it to the `repository/storage/postgres/orm/sequelize/models` because it is a part of the sequelize implementation of storage.